anvil_consortium_manager/views.py

```diff
@@ -177,6 +189,107 @@ def form_valid(self, form):
 
         return super().form_valid(form)
 
+class AccountLink(LoginRequiredMixin, FormView):
+    login_url = settings.LOGIN_URL
+    template_name = "anvil_consortium_manager/account_form.html"
+    model = models.Account
+    message_account_does_not_exist = "This account does not exist on AnVIL."
+    form_class = forms.AccountLinkForm
+
+    def form_valid(self, form):
```

Comment:
Make sure that all the various checks happen in the proper order (not sure what order that should be yet).